UI/src/utils/backend-utils.js

```diff
@@ -619,18 +619,20 @@
 
     this.getGeocode = function(address) {
       var addressNormalized = address.normalize("NFC");
-      var parsedAddress = addressNormalized.split(/^(\s*\w.*)(\s)(\s*\d+\s*),(\s*\w.*)/)
+      var parsedAddress = addressNormalized.split(/^(\s*[A-Za-zÀ-ÿ].*)(\s)(\s*\d+\s*),(\s*[A-Za-zÀ-ÿ].*)/)
         .filter( function(elem) {  return !_.isEmpty(elem.trim()); })
         .map(function(elem) { return elem.trim(); });
 
       return this.getMunicipalityIdByName(parsedAddress[2]).then(
         function(municipalityInfo) {
+          if (_.isEmpty(municipalityInfo))
+            return municipalityInfo;
```

Comment:
Maybe it's a good idea to inform the user of what reason they are not doing anything.

Reply:
It's already doing. If returns empty on municipality or vkm then -> "Tuntematon katuosoite"